docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,1035 @@
+<Timestamp>  INFO  Detected Issuer URL	{"issuer": "https://hypershift-ci-1-oidc.s3.us-east-1.amazonaws.com/jparrill-dest-zjhcn"}
+<Timestamp>  INFO  Created OIDC provider	{"provider": "arn:aws:iam::820196288204:oidc-provider/hypershift-ci-1-oidc.s3.us-east-1.amazonaws.com/jparrill-dest-zjhcn"}
+<Timestamp>  INFO  Created role	{"name": "jparrill-dest-zjhcn-openshift-ingress"}
+<Timestamp>  INFO  Created role policy	{"name": "jparrill-dest-zjhcn-openshift-ingress"}
+<Timestamp>  INFO  Created role	{"name": "jparrill-dest-zjhcn-openshift-image-registry"}
+<Timestamp>  INFO  Created role policy	{"name": "jparrill-dest-zjhcn-openshift-image-registry"}
+<Timestamp>  INFO  Created role	{"name": "jparrill-dest-zjhcn-aws-ebs-csi-driver-controller"}
+<Timestamp>  INFO  Created role policy	{"name": "jparrill-dest-zjhcn-aws-ebs-csi-driver-controller"}
+<Timestamp>  INFO  Created role	{"name": "jparrill-dest-zjhcn-cloud-controller"}
+<Timestamp>  INFO  Created role policy	{"name": "jparrill-dest-zjhcn-cloud-controller"}
+<Timestamp>  INFO  Created role	{"name": "jparrill-dest-zjhcn-node-pool"}
+<Timestamp>  INFO  Created role policy	{"name": "jparrill-dest-zjhcn-node-pool"}
+<Timestamp>  INFO  Created role	{"name": "jparrill-dest-zjhcn-control-plane-operator"}
+<Timestamp>  INFO  Created role policy	{"name": "jparrill-dest-zjhcn-control-plane-operator"}
+<Timestamp>  INFO  Created role	{"name": "jparrill-dest-zjhcn-cloud-network-config-controller"}
+<Timestamp>  INFO  Created role policy	{"name": "jparrill-dest-zjhcn-cloud-network-config-controller"}
+<Timestamp>  INFO  Created role	{"name": "jparrill-dest-zjhcn-worker-role"}
+<Timestamp>  INFO  Created instance profile	{"name": "jparrill-dest-zjhcn-worker"}
+<Timestamp>  INFO  Added role to instance profile	{"role": "jparrill-dest-zjhcn-worker-role", "profile": "jparrill-dest-zjhcn-worker"}
+<Timestamp>  INFO  Created role policy	{"name": "jparrill-dest-zjhcn-worker-policy"}
+<Timestamp>  INFO  Created IAM profile	{"name": "jparrill-dest-zjhcn-worker", "region": "us-west-1"}
+<Timestamp>  INFO  Applied Kube resource	{"kind": "Namespace", "namespace": "", "name": "jparrill"}
+<Timestamp>  INFO  Applied Kube resource	{"kind": "Secret", "namespace": "jparrill", "name": "jparrill-dest-pull-secret"}
+<Timestamp>  INFO  Applied Kube resource	{"kind": "", "namespace": "jparrill", "name": "jparrill-dest"}
+<Timestamp>  INFO  Applied Kube resource	{"kind": "Secret", "namespace": "jparrill", "name": "jparrill-dest-etcd-encryption-key"}
+<Timestamp>  INFO  Applied Kube resource	{"kind": "Secret", "namespace": "jparrill", "name": "jparrill-dest-ssh-key"}
+<Timestamp>  INFO  Applied Kube resource	{"kind": "NodePool", "namespace": "jparrill", "name": "jparrill-dest-us-west-1b"}
+```
+
+</details>
+
+
+After this, the Openshift Hosted Control Plane will continue being deployed and eventually the cluster will create some resources which will end on AWS object creation (These are not managed by Hypershift but Openshift):
```

Comment:
yep, I was talking about the ELB created by the HostedControlPlane as an Ingress ELB. Let me rephrase it to make it clear:

"After this, the Openshift Hosted Control Plane will continue being deployed and eventually it will create some other resources which will end on AWS object creation (ELB) (These are not managed by Hypershift but Openshift):"

WDYT?